src/components/shared/Breed/Breed.js

```diff
@@ -0,0 +1,36 @@
+import {useStyles} from "./BreedStyles";
+import {useDispatch} from "react-redux";
+import {clearImagesReducerAction} from "../../../store/actions/imagesReducerActions";
+import {GridListTile, GridListTileBar} from "@material-ui/core";
+import React from "react";
+import {useHistory} from "react-router-dom";
+import InfoIcon from '@material-ui/icons/Info';
+import {LikeButton} from "../LikeButton";
+
+
+export const Breed = ({breed}) => {
+    const classes = useStyles();
+    const dispatch = useDispatch();
+    let history = useHistory();
+    const handleClick = (id) => {
```

Comment:
You dont need to have id as function parameter, because you can use it direclty from breed.id, e.g history.push(`/gallery/${breed.id}`), and at 28 row use onClick={handleClick}